.github/workflows/unit_test_4gpu.yaml

```diff
@@ -1,42 +1,58 @@
 name: 4 GPU Unit Test
 
+
 on:
   push:
     branches: [ main ]
   pull_request:
 
-concurrency:
-  group: unit-test${{ github.workflow }}-${{ github.ref == 'refs/heads/main' && github.run_number || github.ref }}
-  cancel-in-progress: true
+jobs:
+  build-test:
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.g5.12xlarge.nvidia.gpu
+      gpu-arch-type: cuda
+      gpu-arch-version: "11.6"
```

Comment:
would this be too old for testing, since we are installing the `cu121` nightly version?

Reply:
oh i copypasted this from the generic job template. I will try changing to 12.1.